proposals/lip-0053.md

```diff
@@ -322,11 +322,23 @@ def verifyPartnerChainOutboxRoot(ccu: CCU) -> None:
 
         certificate = decode(certificateSchema, ccu.params.certificate)
         if smtVerifyInclusionProof([outboxKey], proof, certificate.stateRoot) == False:
+            emitPersistentEvent(
+                module = MODULE_NAME_INTEROPERABILITY,
+                name = EVENT_NAME_INVALID_SMT_VERIFICATION,
+                data = {},
+                topic = []
+            )
             raise Exception("Invalid inclusion proof for inbox update.")
 
     # Verification for empty certificates.
     else:
         if newInboxRoot != channel(partnerchainID).partnerChainOutboxRoot:
+            emitPersistentEvent(
+                module = MODULE_NAME_INTEROPERABILITY,
+                name = EVENT_NAME_INVALID_OUTBOX_ROOT,
```

Comment:
As defined earlier in line 1296, "name = EVENT_NAME_INVALID_OUTBOX_ROOT**_VERIFICATION**"

Reply:
fixed